source/examples/rapids-ec2-mnmg/notebook.ipynb

```diff
@@ -63,7 +63,7 @@
    "source": [
     "from dask.distributed import Client\n",
     "\n",
-    "client = Client(cluster)\n",
+    "client = Client(cluster)  # noqa: F821\n",
```

Comment:
`F821` is this:

```text
 F821 Undefined name `cluster`
```

`cluster` will be undefined throughout this notebook because the notebook uses a markdown block with instructions like "go read these other docs to choose how to set up a cluster, then come back here".

<img width="1230" alt="image" src="https://github.com/rapidsai/deployment/assets/7608904/20bf8eb3-a6c5-4fa5-a3dd-8c21c5fcc3c1">

https://github.com/rapidsai/deployment/blob/458febed2c844ff6403a1a43afc2b4744f2116d9/source/examples/rapids-ec2-mnmg/notebook.ipynb#L26-L46

So I think just ignoring the warnings is fine. I chose to do inline `# noqa` comments here so that `ruff` could still catch other "undefined name" problems in this notebook. But if you'd like to keep `# noqa` comments out of these notebooks for readability reasons, I could instead add a rule in `pyproject.toml` telling `ruff` to ignore that check for this file.

Reply:
I think I'd prefer to keep `# noqa` out of the notebooks. I don't want to add complexity for readers just to satisfy a tool. Would you mind doing it in the config?

Reply:
Yep no problem!

Removed those comments and moved this to the config in https://github.com/rapidsai/deployment/pull/370/commits/261a00fa3b6034b20e37d452f42d2666196ff033